packages/desktop/views/dashboard/collectibles/views/CollectiblesDetailsView.svelte

```diff
@@ -155,26 +155,34 @@
             overflow: true,
         })
     }
+
+    function getAlertText(downloadMetadata: NftDownloadMetadata): string {
+        const { error, warning } = downloadMetadata ?? {}
+        const errorOrWarning = error || warning
+
+        if (!errorOrWarning) {
+            return
+        }
+
+        const { type, message } = errorOrWarning
+        return type === 'generic' ? message : localize(`error.nft.${type}.long`)
+    }
 </script>
 
-<div class="flex flex-row w-full h-full space-x-4">
+<collectibles-details-view class="flex flex-row w-full h-full space-x-4">
     <div class="flex w-full h-full items-center justify-center">
         <div class="relative w-full h-full flex rounded-2xl overflow-hidden">
             <NftMedia
                 nftId={id}
-                bind:error
-                bind:warning
                 classes="rounded-2xl overflow-hidden flex-1 w-auto h-auto max-w-full max-h-full object-contain absolute top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2"
                 autoplay
                 controls
                 loop
                 muted
             />
             <div class="absolute right-6 bottom-6 w-auto">
-                {#if error}
-                    <Alert type="error" message={error} />
-                {:else if warning}
-                    <Alert type="warning" message={warning} />
+                {#if alertText}
+                    <Alert type={downloadMetadata?.error ? 'error' : 'warning'} message={alertText} />
                 {/if}
```

Comment:
We should be making this its own component? And splitting up this file in general. Can always add a task to revisit this